### PR TITLE
Tabs on Profiles

### PR DESCRIPTION
**Pull request by Tanza3D**
splits medals into their own tab, allows for extra info without clogging up the main page in the future

### CODE REVIEW

**Comment by Tanza3D** [comment]
probably breaks teams but like, who cares yet

File: global/js/main.js [hunk 1]
```diff
@@ -715,16 +715,44 @@ function getAlerts() {
...
+        return;
+    }
+
+    if (!key) {
+        return;
+    }
+
+    var url = new URL(window.location.href);
+    var params = new window.URLSearchParams(window.location.search);
+    if (value === undefined || value === null) {
```
Comment (EXtremeExploit) [inline global/js/main.js:729]: better to check undefined with typeof
```suggestion
    if (typeof value == 'undefined' || value === null) {

```

**Review by EXtremeExploit — CHANGES_REQUESTED** [review]
one lil thing, but otherwise i think its fine

**Review by EXtremeExploit — APPROVED** [review]
yes

**Pull request merged**